libnavigation-ui/src/main/java/com/mapbox/navigation/ui/internal/utils/ViewUtils.java

```diff
@@ -13,6 +13,9 @@
 
 public class ViewUtils {
 
+  public static final int DEFAULT_BITMAP_ENCODE_WIDTH = 250;
```

Comment:
nit: remove since it's in the builder

Reply:
constants for the object can live in the object. that's where we put it for [NavigationOptions](https://github.com/mapbox/mapbox-navigation-android/blob/fa3a840e92d2f846c4b2cc4467f0183fd2a4cd26/libnavigation-base/src/main/java/com/mapbox/navigation/base/options/NavigationOptions.kt#L13)

could move it to a companion of the object

Reply:
yeah, good catch. I almost forget the two consts are not used any more in this class.